model/file_info_test.go

```diff
@@ -24,109 +24,186 @@ func TestFileInfoIsValid(t *testing.T) {
 		Path:      "fake/path.png",
 	}
 
-	require.Nil(t, info.IsValid())
-
-	info.Id = ""
-	require.NotNil(t, info.IsValid(), "empty Id isn't valid")
-
-	info.Id = NewId()
-	info.CreateAt = 0
-	require.NotNil(t, info.IsValid(), "empty CreateAt isn't valid")
-
-	info.CreateAt = 1234
-	info.UpdateAt = 0
-	require.NotNil(t, info.IsValid(), "empty UpdateAt isn't valid")
-
-	info.UpdateAt = 1234
-	info.PostId = NewId()
-	require.Nil(t, info.IsValid())
-
-	info.Path = ""
-	require.NotNil(t, info.IsValid(), "empty Path isn't valid")
-
-	info.Path = "fake/path.png"
-	require.Nil(t, info.IsValid())
+	t.Run("Original File Info", func(t *testing.T) {
```

Comment:
I would name it Valid file info, instead of original

Reply:
Should be corrected at 6fa5062 along with the other comments.

Thanks!